data/content_scripts/posting_process/sendkeys.js

```diff
@@ -0,0 +1,766 @@
+// Cross-broswer implementation of text ranges and selections
+// documentation: http://bililite.com/blog/2011/01/17/cross-browser-text-ranges-and-selections/
+// Version: 2.6
+// Copyright (c) 2013 Daniel Wachsstock
+// MIT license:
```

Comment:
This is a rather large lib to inject. Is there something that the prior series of events was not solving? Is this only injected as a content script when the posting process is initiated? Otherwise this will have a pretty big footprint across all windows and iframes.